.gitignore

```diff
@@ -15,6 +15,7 @@ locale/
 
 # Windows build
 build/
+cleanerml/
```

Comment:
This looks like it doesn't belong

Reply:
This is for making automated BetaTester & AlphaTester Builds (that include the cleaners from "release" & "pending" in installation, as own topic/sub-section). I can program in NSIS the direct path to the CleanerML Repo, but unfortunately I can relative just go one path level higher and not two. (Can just use "..\cleanerml" and not "...\cleanerml".)